app/views/operational_fields/show.html.erb

```diff
@@ -1,28 +1,25 @@
 <% page_title @operational_field.name %>
 <% page_class "operational-field-show" %>
 
-<header class="block headings-block">
-  <div class="inner-block floated-children">
-    <%= render partial: 'shared/heading',
-      locals: { type: 'British fatalities',
-      heading: "Operations in #{@operational_field.name}",
-      big: true, extra: true
-    } %>
-    <div class="heading-extra">
-      <div class="inner-heading">
-        <div class="organisations-icon-list">
-          <%= content_tag_for(:div, @organisation,
-            class: organisation_brand_colour_class(@organisation)) do %>
-            <%= link_to @organisation,
-            class: logo_classes(organisation: @organisation, stacked: true, use_identity: true) do %>
-              <span><%= organisation_logo_name(@organisation) %></span>
-            <% end %>
-          <% end %>
-        </div>
-      </div>
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= render "govuk_publishing_components/components/title", {
+        context: "British fatalities",
+        title: "Operations in #{@operational_field.name}",
+      } %>
+    </div>
+    <div class="govuk-grid-column-one-third">
+      <%= content_tag_for(:div, @organisation,
```

Comment:
Out of scope for the ticket but this could possibly be an [org logo component](https://components.publishing.service.gov.uk/component-guide/organisation_logo). It's a bit of a tricky component so no worries if it turns into a bother.

Reply:
Agreed. I'm mindful to keep this in scope - I have captured though.